docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -739,30 +739,136 @@ Authenticated API, `organization_config`:
             4
         ],
         "req": {
-            "cmd": "organization_config"
+            "cmd": "shamir_recovery_delete_setup",
+            "fields": [
+                {
+                    "name": "certificate",
+                    // Contains a ShamirRecoveryDeletionCertificate
+                    "type": "bytes"
+                }
+            ]
         },
         "reps": [
             {
-                "status": "ok",
+                "status": "ok"
+            },
+            {
+                // Cannot deserialize data into the expected certificate
+                "status": "invalid_data"
+            },
+             {
+                // Share recipients lists are incoherent
+                "status": "incoherent_date"
```

Comment:
Share recipients are the only not identifying data that we have in the deletion certificate to check. Maybe we could include more data from the brief to ensure coherence. I'm not sure how much it's needed though